lib/core/lib/ops/teamOps.js

```diff
@@ -286,7 +286,7 @@ function createDefaultUser (job, next) {
   const { organizationId } = job.params
 
   userOps.insertUser(job.client, { id, name, organizationId }, (err, user) => {
-    if (err) return next(Boom.badImplementation(err))
+    if (err) return next(err)
```

Comment:
I had to change that as it was blowing up hapi, as I recall it the err was already wrapped in Boom (coming from `insertUser`) and was being re-wrapped here.